src/test/java/no/unit/nva/model/instancetypes/TestData.java

```diff
@@ -0,0 +1,51 @@
+package no.unit.nva.model.instancetypes;
+
+import no.unit.nva.model.exceptions.InvalidPageRangeException;
+import no.unit.nva.model.pages.Range;
+
+import java.util.Random;
+
+public class TestData {
+    protected static final Random RANDOM = new Random();
+
+    private final String begin;
+    private final String end;
+    private boolean peerReviewed;
+
+    /**
+     * Creates a random set of data for the Range that is present in each Pages subtype.
+     */
+    public TestData() {
+        begin = random(1, 12);
```

Comment:
I do not understand the necessity of random data. Is there a case where a specific combination might throw an Exception? Then this combination should be tested.  Is it to ensure that values from other tests are not surviving and some tests are passing due to surviving values between tests? This could mean that we should simplify our tests.